purchases/src/main/kotlin/com/revenuecat/purchases/CustomerInfoUpdateHandler.kt

```diff
@@ -19,6 +20,7 @@ internal class CustomerInfoUpdateHandler(
     private val identityManager: IdentityManager,
     private val offlineEntitlementsManager: OfflineEntitlementsManager,
     private val handler: Handler = Handler(Looper.getMainLooper()),
+    private val appConfig: AppConfig,
```

Comment:
Nitpick, but I prefer to have parameters without default values before those with default values.